### PR TITLE
INFRA-126 - fix an error that kills pods [Proof of concept]

### PR DESCRIPTION
**Pull request by toplenboren**

### CODE REVIEW

**Review by sitozzz — APPROVED** [review]

File: apps/condo/domains/common/utils/sberCloudFileAdapter.js [hunk 1]
```diff
@@ -229,25 +231,30 @@ const obsRouterHandler = ({ keystone }) => {
...
-            // check property access case
-            if (hasAccessToReadFile && isString(propertyQuery) && !isNil(propertyValue)) {
-                const propertyPath = propertyQuery
-                    .replaceAll('}', '') // remove close brackets of sub props querying
-                    .split('{') // work with each path parts separately
-                    .map(path => path.trim()) // since gql allow to have spaces in querying - let's remove them
-                    .join('.') // join by . for lodash get utility
-                hasAccessToReadFile = get(item, propertyPath) == propertyValue
+            try {
+                const item = await getItem({
```
Comment (pahaz) [inline apps/condo/domains/common/utils/sberCloudFileAdapter.js:237]: it looks like you don't need to check access here. Use `getByID` util here

Comment (toplenboren) [reply apps/condo/domains/common/utils/sberCloudFileAdapter.js:237]: No, I think this code was supposed to check access by design

Comment (toplenboren) [reply apps/condo/domains/common/utils/sberCloudFileAdapter.js:237]: Gonna look into it :/

Comment (pahaz) [reply apps/condo/domains/common/utils/sberCloudFileAdapter.js:237]: We also have execute gql by user. It looks like we need to use it here?!

**Pull request closed without merging**